ci-operator/step-registry/interop/opp/preflight/interop-opp-preflight-commands.sh

```
@@ -11,59 +10,61 @@ export XDG_RUNTIME_DIR="${HOME}/run"
 export REGISTRY_AUTH_PREFERENCE=podman
 mkdir -p "${XDG_RUNTIME_DIR}"
 
-if ! command -v jq &>/dev/null; then
-    echo "jq not found; installing..."
-    dnf install -y -q jq 2>/dev/null || yum install -y -q jq 2>/dev/null || {
-        echo >&2 "ERROR: failed to install jq"
-        exit 1
-    }
+if [[ -f "${SHARED_DIR}/proxy-conf.sh" ]]; then
+    set +x
+    source "${SHARED_DIR}/proxy-conf.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./proxy-conf.sh was not specified as input (see shellcheck -x).

(SC1091)